### PR TITLE
replace use of master with main

### PR DESCRIPTION
**Pull request by jedwards4b**
### Description of changes
Replace usage of master with main, both with reference to github branch and primary task in fortran code. 

### Specific notes

Contributors other than yourself, if any:

CMEPS Issues Fixed (include github issue #):

Are changes expected to change answers? (specify if bfb, different at roundoff, more substantial) 

Any User Interface Changes (namelist or namelist defaults changes)?

### Testing performed

Testing performed if application target is CESM:
- [ ] (recommended) CIME_DRIVER=nuopc scripts_regression_tests.py
   - machines:
   - details (e.g. failed tests):
- [ ] (recommended) CESM testlist_drv.xml
   - machines and compilers:
   - details (e.g. failed tests):
- [ ] (optional) CESM prealpha test
   - machines and compilers
   - details (e.g. failed tests):
- [ ] (other) please described in detail
   - machines and compilers
   - details (e.g. failed tests):

Testing performed if application target is UFS-coupled:
- [ ] (recommended) UFS-coupled testing
   - description:
   - details (e.g. failed tests):

Testing performed if application target is UFS-HAFS:
- [ ] (recommended) UFS-HAFS testing
   - description:
   - details (e.g. failed tests):

### Hashes used for testing:

- [ ] CESM:
  - repository to check out: https://github.com/ESCOMP/CESM.git
  - branch/hash:
- [ ] UFS-coupled, then umbrella repostiory to check out and associated hash:
  - repository to check out:
  - branch/hash:
- [ ] UFS-HAFS, then umbrella repostiory to check out and associated hash:
  - repository to check out:
  - branch/hash:

### CODE REVIEW

File: cesm/driver/ensemble_driver.F90 [hunk 1]
```diff
@@ -9,7 +9,7 @@ module Ensemble_driver
 
   use shr_kind_mod  , only : cl=>shr_kind_cl, cs=>shr_kind_cs
   use shr_log_mod   , only : shr_log_setLogUnit
-  use esm_utils_mod , only : mastertask, logunit, chkerr
+  use esm_utils_mod , only : maintask, logunit, chkerr
```
Comment (uturuncoglu) [inline cesm/driver/ensemble_driver.F90:12]: It seems that you converted all master->main. Did you do it intentionally or just run a script to replace them. Just wonder. It is okay for me if you replay want to change master to main in everywhere

**Review by uturuncoglu — APPROVED** [review]
@jedwards4b do we need to test it under UFS? It seems it is just replacing master with main and if it compiles under CESM, it will probably works under UFS too.

File: cesm/driver/ensemble_driver.F90 (hunk 1 above)
Comment (jedwards4b) [reply cesm/driver/ensemble_driver.F90:12]: Yes, I just used a script, but my intent was to replace master with main in all cases.  Except possibly in OpenMP directives which as far as I can tell still use master slave language.

Comment (uturuncoglu) [reply cesm/driver/ensemble_driver.F90:12]: Okay. That sounds fine. Thanks.

**Pull request merged**